tests/acceptance/tests/Visual/Administration/Promotions.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { test, expect, setViewport, replaceElements } from '@fixtures/AcceptanceTest';
+
+test('Visual: Promotions Listing Page', { tag: '@Visual' }, async ({ 
+    ShopAdmin, 
+    TestDataService, 
+    AdminPromotionsListing,
+ }) => {
+    await test.step('Creates a screenshot of the promotions listing page in its empty state.', async () => { 
+        await ShopAdmin.goesTo(AdminPromotionsListing.url());
+        await setViewport(AdminPromotionsListing.page, {
```

Comment:
Please always use the requestURL or waitForElement arguments to significantly speed up the test. In this case, `requestURL: 'api/search/promotion',` works fine and shaves 5-6 seconds off of the overall test duration.